src/Components/Cluster/Cluster.cy.js

```diff
@@ -1,231 +1,188 @@
 import React from 'react';
-import { mount } from '@cypress/react';
-import { MemoryRouter } from 'react-router-dom';
-
-import { Intl } from '../../Utilities/intlHelper';
-import { Cluster } from './Cluster';
-import { Provider } from 'react-redux';
-import getStore from '../../Store';
+import Cluster from '.';
 import singleClusterPageReport from '../../../cypress/fixtures/api/insights-results-aggregator/v2/cluster/dcb95bbf-8673-4f3a-a63c-12d4a530aa6f/reports-disabled-false.json';
 import {
   checkNoMatchingRecs,
   checkRowCounts,
 } from '../../../cypress/utils/table';
-import FlagProvider from '@unleash/proxy-client-react';
 
 // selectors
 const CLUSTER_HEADER = '#cluster-header';
 const BREADCRUMBS = 'nav[class=pf-c-breadcrumb]';
 const RULES_TABLE = '#cluster-recs-list-table';
 const FILTER_CHIPS = 'li[class=pf-c-chip-group__list-item]';
-let props;
 
-describe('cluster page', () => {
-  beforeEach(() => {
-    // the flag tells not to fetch external federated modules
-    window.CYPRESS_RUN = true;
-
-    props = {
-      cluster: {
-        isError: false,
-        isUninitialized: false,
-        isLoading: false,
-        isFetching: false,
-        isSuccess: true,
-        data: singleClusterPageReport,
-      },
-      displayName: {
-        data: singleClusterPageReport.report.meta.cluster_name,
-      },
-      clusterId: 'foobar',
-    };
-  });
+// interceptors
```

Comment:
Intercepting network requests instead of mocking props helps to create more abstraction from the implementation details.